apps/fhir/bluebutton/migrations/0002_archivedcrosswalk.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 2.2.24 on 2021-07-02 17:59
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('bluebutton', '0001_squashed_0006_auto_20210513_1812'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='ArchivedCrosswalk',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('_fhir_id', models.CharField(db_column='fhir_id', db_index=True, default=None, max_length=80)),
+                ('user_id_type', models.CharField(choices=[('H', 'HICN'), ('M', 'MBI')], default='H', max_length=1, verbose_name='Hash ID type last used for FHIR_ID lookup')),
+                ('_user_id_hash', models.CharField(db_column='user_id_hash', db_index=True, default=None, max_length=64, verbose_name='HASH of User HICN ID')),
+                ('_user_mbi_hash', models.CharField(db_column='user_mbi_hash', db_index=True, default=None, max_length=64, null=True, verbose_name='HASH of User MBI ID')),
+                ('archived_at', models.DateTimeField(auto_now_add=True)),
+                ('crosswalk', models.ForeignKey(blank=True, db_constraint=False, null=True, on_delete=django.db.models.deletion.CASCADE, to='bluebutton.Crosswalk')),
```

Comment:
Is the foreign key necessary you think?

Reply:
> Is the foreign key necessary you think?

You're right. This is not needed. Will update the code shortly.

Reply:
Thinking on this more - it's basically if we want the archive to outlast the crosswalk record. Like if the crosswalk record is deleted, do we want the archived/audit trail to still exist for it? I'm thinking yes? Maybe we'll create some nightly job to eventually clean up the archived table but for now I think keeping the data around is good.

Reply:
> Thinking on this more - it's basically if we want the archive to outlast the crosswalk record. Like if the crosswalk record is deleted, do we want the archived/audit trail to still exist for it? I'm thinking yes? Maybe we'll create some nightly job to eventually clean up the archived table but for now I think keeping the data around is good.

Yes to this :+1: